lesson07/assignment/assignment07/src/views/About.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <div class="about">
+    
+    <AboutPage />
```

Comment:
For all these pages, if they just contain a single component, there's no need for it to be a component- all the code from it could be in this view file. Components are best used for small, reusable chunks of code. For example, the prices content within the accordion on Menu page might be a component. The way you have broken all page content into components works, just isn't best practice.